package.json

```diff
@@ -101,6 +101,21 @@
           "default": true,
           "description": "Ask to select a Meson project root directory when more than one project is detected."
         },
+        "mesonbuild.setupCppTools": {
+          "type": "boolean",
+          "default": true,
+          "description": "Setup the cpptools extension to work with Meson."
+        },
+        "mesonbuild.setupClangd": {
+          "type": "boolean",
+          "default": true,
+          "description": "Setup the vscode-clangd extension to work with Meson."
+        },
+        "mesonbuild.setupRustAnalyzer": {
+          "type": "boolean",
+          "default": true,
+          "description": "Setup the rust-analyzer extension to work with Meson."
+        },
```

Comment:
My thoughts were that you may want to have both the C++ and clangd extensions installed, but you may want to use only one of them for a particular project. There are also other considerations, for example, you may be fine with updating the C++ extension configuration, which goes through a dedicated setting, but you may not be fine with modifying the clangd arguments list, which is passed to an external executable.

I don't have a strong preference for one option or the other, but with the changes @tristan957 made, we explicitly tell users that we have support for this and that extension. This would go away if there was only one option to choose from.

What are your thoughts on this?